src/ngircd/client.c

```diff
@@ -38,7 +38,7 @@
 #include "match.h"
 #include "messages.h"
 
-#define GETID_LEN (CLIENT_NICK_LEN-1) + 1 + (CLIENT_USER_LEN-1) + 1 + (CLIENT_HOST_LEN-1) + 1
+#define GETID_LEN (CLIENT_ID_LEN-1) + 1 + (CLIENT_USER_LEN-1) + 1 + (CLIENT_HOST_LEN-1) + 1
```

Comment:
`CLIENT_ID_LEN = 64` while `CLIENT_NICK_LEN = 32`. `GETID_LEN` is used as a size of the array here https://github.com/ngircd/ngircd/blob/02a572d829e372eb32b3d4187b43e1dc9a553283/src/ngircd/client.c#L928 and the first argument is `Client->id`. Going to the header file here https://github.com/ngircd/ngircd/blob/02a572d829e372eb32b3d4187b43e1dc9a553283/src/ngircd/client.h#L44, the size of the field can be a maximum of `CLIENT_ID_LEN`, and the comment states that it could either hold a user nick or a server ID. Thus, my change should be correct to allow for the maximum size.

This fixes the (correctly caught) compiler warning.